src/vm/arm64/virtualcallstubcpu.hpp

```diff
@@ -217,8 +217,8 @@ struct ResolveHolder
          //w13- this._hashedToken
          //ldr w13, [x10 + DATA_OFFSET(_hashedToken)]
          offset = DATA_OFFSET(_hashedToken);
-         _ASSERTE(offset >=0 && offset%8 == 0);
-         _stub._resolveEntryPoint[n++] = 0xB940014D | offset<<7;
+         _ASSERTE(offset >=0 && offset%4 == 0);
+         _stub._resolveEntryPoint[n++] = 0xB940014D | offset<<8;
```

Comment:
Since we are loading a 4byte address, the offset needs to be (offset >> 2) << 10.  As opposed to (offset >> 3) << 10 for 8 byte loads below.